scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -101,14 +108,15 @@ task ExtractTask {
         String fq_cohort_extract_table
         String read_project_id
         String output_file
-        String fq_filter_set_info_table
-        String fq_filter_set_site_table
-        String fq_filter_set_tranches_table
-        String filter_set_name
-        Float? snps_truth_sensitivity_filter_level
-        Float? indels_truth_sensitivity_filter_level
+        String? output_gcs_dir
 
         Boolean do_not_filter_override
+        String? fq_filter_set_info_table
+        String? fq_filter_set_site_table
+        String? fq_filter_set_tranches_table
```

Comment:
Anything that is assigned a default value in the outer workflow should not be optional here.
Also, filter_set_name should not be optional